### PR TITLE
Fix only test eBGP neighbors for test_bgp_queues

### PR DESCRIPTION
**Pull request by auspham**
### Description of PR

Summary: only test eBGP neighbors for test_bgp_queues
Fixes # (issue) 

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
Complete #11116 

#### How did you verify/test it?
T2 profile

### CODE REVIEW

**Review by sdszhang — APPROVED** [review]
LGTM

File: tests/bgp/test_bgp_queue.py [hunk 1]
```diff
@@ -59,6 +59,10 @@ def test_bgp_queues(duthosts, enum_frontend_dut_hostname, enum_asic_index, tbinf
     for k, v in list(bgp_facts['bgp_neighbors'].items()):
         # Only consider established bgp sessions
         if v['state'] == 'established':
+            # For "peer group" if it's internal it will be "INTERNAL_PEER_V4" or "INTERNAL_PEER_V6"
```
Comment (yejianquan) [inline tests/bgp/test_bgp_queue.py:62]: Could you also leave the sample value when it's external BGP peers?

Comment (auspham) [reply tests/bgp/test_bgp_queue.py:62]: Sure I've added one for external

**Review by yejianquan — APPROVED** [review]
LGTM

**Pull request merged**